### PR TITLE
Fix #1721

### LINKED ISSUE #1721: CmpLog and autotokens borrow https://github.com/AFLplusplus/AFLplusplus/issues/1927 from AFL++
https://github.com/AFLplusplus/LibAFL/blob/c2db7ad1623fceb047b3a22e51e0a91750ffc312/libafl_cc/src/cmplog-switches-pass.cc#L113

https://github.com/AFLplusplus/LibAFL/blob/main/libafl_cc/src/autotokens-pass.cc#L130

### PR DESCRIPTION
**Pull request by andreafioraldi**

### CODE REVIEW

File: libafl_cc/src/cmplog-switches-pass.cc [hunk 1]
```diff
@@ -233,7 +166,7 @@ bool CmpLogSwitches::hookInstrs(Module &M) {
   }
 
   for (auto &F : M) {
-    if (!isIgnoreFunction(&F)) { continue; }
```
Comment (andreafioraldi) [inline libafl_cc/src/cmplog-switches-pass.cc:236]: @tokatoka I reversed this not, it seems it was broken, can you check?

Comment (tokatoka) [reply libafl_cc/src/cmplog-switches-pass.cc:236]: yes you are correct

File: libafl_cc/src/autotokens-pass.cc [hunk 2]
```diff
@@ -33,18 +33,11 @@
 #include <fstream>
 #include <set>
 
-#include "llvm/Config/llvm-config.h"
+#include "common-llvm.h"
+
 #include "llvm/ADT/Statistic.h"
 #include "llvm/IR/IRBuilder.h"
 
-#if USE_NEW_PM
```
[the remark this answers is not in the document]
Comment (tokatoka) [reply libafl_cc/src/autotokens-pass.cc:40]: it makes older llvm broken

[the remark this answers is not in the document]
Comment (tokatoka) [reply libafl_cc/src/autotokens-pass.cc:40]: ah ok nevermind
there's the same include in common-llvm.h

**Pull request merged**